pyscriptjs/src/utils.ts

```diff
@@ -109,3 +111,35 @@ export function getAttribute(el: Element, attr: string): string | null {
     }
     return null;
 }
+
+export function calculatePaths(fetch_cfg: FetchConfig[]) {
+    const fetchPaths: string[] = [];
+    const paths: string[] = [];
+    fetch_cfg.forEach(function (each_fetch_cfg: FetchConfig) {
+        const from = each_fetch_cfg.from || "";
+        const to_folder = each_fetch_cfg.to_folder || ".";
+        const files = each_fetch_cfg.files;
```

Comment:
this is missing support for `to_file` as described here: https://github.com/pyscript/pyscript/issues/519#issuecomment-1302362956

Reply:
@antocuni I think you reviewed a previous commit. I already added support for `to_file`. Not sure why you cannot see it.

Reply:
resolving...